tokio/src/runtime/builder.rs

```diff
@@ -375,6 +385,30 @@ impl Builder {
             blocking_pool,
         })
     }
+
+    #[cfg(feature = "blocking")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "blocking")))]
+    /// Sets a custom timeout for a thread in the blocking pool.
+    /// By default, the timeout for a thread is constant and defined
+    /// in KEEP_ALIVE. In case the user wants to customize this
```

Comment:
Is the `KEEP_ALIVE` constant publicly exported anywhere? I don't believe it is. Since it's not public, I'd prefer to just state what its value is.